charts/certificate-manager/templates/cluster-issuer.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: cert-manager.io/v1alpha2
+kind: ClusterIssuer
```

Comment:
Thanks for pointing this out. Makes sense. Additionally, a `ClusterIssuer` doesnt even live in a namespace. Tho, then it could be even challenged as to why creating a separate chart in the first place? What do you think?

Reply:
We can just use an [`Issuer`](https://cert-manager.io/docs/concepts/issuer/) instead of  `ClusterIssuer`. `Issuer` will have a namespace and won't react to certificates/annotations in other namesapces

Reply:
Sorry, I was misleading in my Question. I'm aware of the `Issuer` kind. I was rather wondering what is your opinion on the separate chart part I mentioned - whether we should keep it or move the `Issuer` into the ingress services charts?

Reply:
we ended up going with an `Issuer` instead of a `ClusterIssuer` and moved it into the ingress services chart.